test/recipes/30-test_evp_data/evpciph.txt

```diff
@@ -1191,12 +1191,21 @@ Result = CIPHERFINAL_ERROR
 Title = AES XTS test vectors from IEEE Std 1619-2007
 
 Cipher = aes-128-xts
+Operation = ENCRYPT
 Key = 0000000000000000000000000000000000000000000000000000000000000000
 IV = 00000000000000000000000000000000
 Plaintext = 0000000000000000000000000000000000000000000000000000000000000000
 Ciphertext = 917cf69ebd68b2ec9b9fe9a3eadda692cd43d2f59598ed858c02c2652fbf922e
 Result = KEY_SET_ERROR
 
+# This is banned under FIPS
```

Comment:
It might be nice to say what specifically is banned (i.e. both keys are the same)

Reply:
Added some additional comments.
I'm not intending to add the FIPS mode support at this point, I don't believe that a definitive decision as to how this will be done has yet been made.